### ISSUE TITLE
Request for a way to see what scholars are researching, as well as individual progress towards their goal.

### ISSUE THREAD

**Issue by BeerNTacos** [DFHack/dfhack#3443]
Per a conversation I had with /u/Ayaire on reddit, would it be possible to create a script to check on the progress a scholar is making on their research subject? To add to that, it would be even more useful/interesting to see what each of a fortress' scholars are researching, as well as their individual XP progress for the subject matter they are each researching.

From some tinkering, I believe that those who have been scholars before still have an existing XP level, so perhaps the locals on the map who are not at the time scholars but have experience in it could have that data displayed as well?

Enabling a user to be able to keep track of their scholar's progress may help with their personnel planning, plus people always like to see numbers advance.

**Comment by myk002** [comment]
Sounds like a good idea. What would we call such a tool?

**Comment by BeerNTacos** [comment]
> Sounds like a good idea. What would we call such a tool?

@myk002 Hmm, if we were to name the command after what it does, something like "scholar_research_status" would be most apt, but that may be a bit wordy. "Scholars" may be an issue if other commands involving scholars show up in the future, but commands can always be changed.

Now that I'm thinking about it, the "scholars" command could have different command line switches to parse out what's shown with the command, so maybe default could show just what the current active in-house and/or visiting scholars are studying in the fortress and their XP progress, while other switches could show active in-house and/or visiting entities with historical scholar XP, plus, just a sudden thought, another switch perhaps showing the books a scholar has written in the past?

Letting a user know that their fortress members have written other books that are out in the world can give that user an excuse to go artifact hunting.
